docs/api-markdown-documenter/render-api-documentation.js

```diff
@@ -105,9 +108,6 @@ export async function renderApiDocumentation(inputDir, outputDir, uriRootDir, ap
 			// TODO: Also skip `@fluid-internal` packages once we no longer have public, user-facing APIs that reference their contents.
 			return ["@fluid-private"].includes(packageScope);
 		},
-		// Temporary workaround to ensure APIs temporarily tagged as `@alpha` (to mean "legacy") are included in the API docs.
-		// This min level should be set back to Beta once legacy APIs have been cleaned up to not use `@alpha`.
-		minimumReleaseLevel: ReleaseTag.Alpha, // Don't include `@internal` items in docs published to the public website.
```

Comment:
You showed an example of devtools-core `@alpha` use, but will those show with this removed? Example just in case other level is used in some situation?

Reply:
This is actually the default behavior. It was originally set to `Beta`, but I had to put it back to `Alpha` to ensure legacy things showed up on the website. Now that `legacy` and `alpha` are detangled, we can include `alpha` things on the website again, with their correct meaning.

This does mean that all `alpha` items will now be visible on the website, but I think that's okay given that there is a meaningful support contract associated with them now.

Reply:
I was wondering what I should do for peeps wanting to play with alpha APIs. Having docs seems like the good plan to me. They don't have to read 'em if they don't want to. :)

Reply:
Yea, with alpha reclaimed, making it visible on our website/docs makes sense.